docs/source/contribution/development_for_databricks.md

```diff
@@ -1,13 +1,11 @@
-# How to deploy a development version of Kedro to Databricks
+# Contribute changes to Kedro that are tested on Databricks
 
-[Databricks](https://www.databricks.com/) is a cloud-based platform for data engineering and data science that many Kedro users deploy their Kedro projects to. Extending and improving the Kedro experience on Databricks is important to a large part of Kedro's user base.
+Many Kedro users deploy their projects to [Databricks](https://www.databricks.com/), a cloud-based platform for data engineering and data science. We encourage contributions to extend and improve the experience for Kedro users on Databricks; this guide explains how to efficiently test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
 
-## Introduction
-
-This guide describes how to efficiently develop features and fixes for Kedro on Databricks. Using this guide, you will be able to quickly test your locally modified version of Kedro on Databricks as part of a build-and-test development cycle.
+## How to deploy a development version of Kedro to Databricks
```

Comment:
```suggestion
## Deploy a development version of Kedro to Databricks
```

For consistency with the rest of our titles, this might be the better way.